__tests__/pathfinding.service.test.js

```diff
@@ -1,6 +1,6 @@
 import * as testData from "./__testData__/pathfinding.data";
-import { POIInfo } from "../src/constants/poi.data";
-import * as floors from "../src/constants/floors.data";
+import { POIInfo } from "../src/constants";
+import * as floors from "../src/constants";
```

Comment:
From what I can see, only the field `buildingFloors` from `floors`. consider changing this to:
`import { POIInfo, buildingFloors } from "../src/constants";` and replace all instances of `floors.buildingFloors` to `buildingFloors`

Reply:
Fixed

Reply:
Fixed